data/scripts/creaturescripts/northern_pike_death.lua

```diff
@@ -0,0 +1,10 @@
+local creatureevent = CreatureEvent("NorthernPikeDeath")
+
+function creatureevent.onDeath(creature, corpse, lasthitkiller, mostdamagekiller, lasthitunjustified, mostdamageunjustified)
+	if math.random(100) < 11 then
+		Game.createMonster("Slippery Northern Pike", creature:getPosition())
```

Comment:
I think you could put there force parameter so it always summons that monster in place.
so false, true after getPosition()